waku/waku_test.go

```diff
@@ -484,12 +475,16 @@ func TestExpiry(t *testing.T) {
 		t.Fatal("failed to set min pow")
 	}
 
-	defer w.SetMinimumPoW(common.DefaultMinimumPoW, false) // nolint: errcheck
+	defer func() {
+		handleError(t, w.SetMinimumPoW(common.DefaultMinimumPoW, false))
+	}()
 	err = w.Start(nil)
 	if err != nil {
 		t.Fatal("failed to start waku")
 	}
-	defer w.Stop() // nolint: errcheck
+	defer func() {
+		handleError(t, w.Stop())
+	}()
```

Comment:
[Error handling is good](https://github.com/status-im/status-go/pull/1957/files#r417328424)